plyer/platforms/macosx/libs/osx_paths.py

```diff
@@ -1,11 +1,15 @@
 import ctypes
 import os
 
+LibraryPath = \
+    "/System/Library/Frameworks/CoreFoundation.framework/\
+    Versions/A/CoreFoundation"
```

Comment:
i think this is a breaking change here, the spaces will be part of the string, you could use something like
```
LibraryPath = ''.join(
    "/System/Library/Frameworks/CoreFoundation.framework/",
   "Versions/A/CoreFoundation"
)
```

or even simply
```
LibraryPath = (
    "/System/Library/Frameworks/CoreFoundation.framework/"
    "Versions/A/CoreFoundation"
)
```
(note the lack of comma at the end of the line, so strings are simply concatenated)